src/Aspire.Hosting/ResourceBuilderExtensions.cs

```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Data;
```

Comment:
Why is this namespace needed?

Reply:
Not sure but the build would fail if it wasn't required.

Reply:
Not sure why the analyzer isn't picking this one up but you're right, it isn't needed.